package.nls.json

```diff
@@ -430,8 +430,8 @@
     "DataScience.jupyterSelectURIQuickPickTitle": "Pick how to connect to Jupyter",
     "DataScience.jupyterSelectURIQuickPickPlaceholder": "Choose an option",
     "DataScience.jupyterSelectURIQuickPickCurrent": "Current: {0}",
-    "DataScience.jupyterSelectURILocalLabel": "Default",
-    "DataScience.jupyterSelectURILocalDetail": "VS Code will automatically start a server for you on the localhost",
+    "DataScience.jupyterSelectURINoneLabel": "None",
+    "DataScience.jupyterSelectURINoneDetail": "Do not connect to any remote Jupyter server",
```

Comment:
Is this just a no-op? It sounds more like 'cancel', but don't we usually just let the user close the quick-pick for that?

Reply:
never mind, I just tried it out and see that it drops the remote connections from the kernel picker - that description makes sense then